src/components/DeckCard.js

```diff
@@ -114,17 +116,18 @@ const DeckCard = ({ seat, deck, seatIndex, view }) => {
       </CardHeader>
       {view === 'picks' && (
         <CardBody>
-          {deck.type === 'd' ? (
-            <>
-              {deck.seats[0].pickorder ? (
+          {
+            // eslint-disable-next-line no-nested-ternary
```

Comment:
`no-nested-ternary` is another rule that I feel like should be disabled at least in JSX code, since you don't have other options there.